core/src/main/scala/kafka/metrics/KafkaMetricsReporter.scala

```diff
@@ -52,12 +52,11 @@ trait KafkaMetricsReporter {
 }
 
 object KafkaMetricsReporter {
-  val ReporterStarted: AtomicBoolean = new AtomicBoolean(false)
```

Comment:
We didn't remove this when cluster id was added because `KafkaMetricsReporter` is a pluggable interface and it wasn't clear to me if external users were using `KafkaMetricsReporter.ReporterStarted`. In other words, I assumed this class to be public since `KafkaMetricsReporter` is public.